README_BEAGLEBONE_BLACK.md

```diff
@@ -0,0 +1,162 @@
+
+### Step 4: Compile and install SuperCollider
+
+    cd ~ # or cd into the directory where you'd like to clone and build supercollider
+    git clone --recurse-submodules https://github.com/supercollider/supercollider.git
+    cd supercollider
+    mkdir build && cd build
+    cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DSUPERNOVA=OFF -DNATIVE=ON -DSC_IDE=OFF -DNO_X11=ON \
+        -DSC_QT=OFF -DSC_ED=OFF -DSC_EL=OFF -DSC_VIM=ON ..
+    cmake --build . --config Release --target all
+    sudo cmake --build . --config Release --target install
+    sudo ldconfig
+
+
+Usage
+-----
+
+To use SuperCollider, just open a terminal and execute `sclang`.
+
+**Done!** See below for other usage notes and tips.
+
+sc3-plugins
+-----------
+
+To compile and install sc3-plugins, follow the instructions in the [sc3-plugins README](https://github.com/supercollider/sc3-plugins).
+
+Autostart
+---------
+
+To automatically run SuperCollider code at system boot:
+
+    cat >~/autostart.sh <<EOF
+    #!/bin/bash
+    export PATH=/usr/local/bin:$PATH
+    export DISPLAY=:0.0
+    sleep 10 # can be lower (5) for rpi3
+    sclang mycode.scd
+    EOF
+
+    chmod +x ~/autostart.sh
+    crontab -e # and add the following line to the end:
+        @reboot cd /home/pi && ./autostart.sh
+
+    nano ~/mycode.scd # And add your code inside a waitForBoot. For example:
+        s.waitForBoot{ {SinOsc.ar([400, 404])}.play }
+
+    sudo reboot # the sound should start after a few seconds
+
+Login with ssh and run `killall jackd sclang scsynth` to stop the sound.
+
+Benchmarks
+----------
+
+These are rough benchmark tests. The server should be booted and jackd running with settings: `-P75 -p1024 -n3 -r44100`
+
+Also make sure to power the bbb through the barrel jack (else the CPU is capped at 300mhz - check with
+`cpufreq-info`). Also for comparison it is important to set CPU scaling to 'performance' with:
+
+    sudo cpufreq-set --governor performance`
+
+Also for comparison it is important to set CPU scaling to 'performance', by running:
+
+    echo performance | sudo tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor
+
+Start sclang or scide and run:
+
+    s.boot
+    {1000000.do{2.5.sqrt}}.bench // ~1.9 on a bb-black, ~2.6 with lxqt on a bb-black
+    a= {Mix(50.collect{RLPF.ar(SinOsc.ar)});DC.ar(0)}.play
+    s.avgCPU // run a few times. ~47% on a bb-black
+    a.free
+
+The default CPU scaling mode 'ondemand' seem to work quite good on the BBB and will save battery without losing much
+in performance.
+
+Notes and Troubleshooting
+-------------------------
+
+This applies to both GUI and GUI-less builds above:
+
+* An easy way to burn the img.xy file (no need to unpack) to an SD card is to use [etcher](http://etcher.io).
+* The internal soundcard volume is by default set low (40). Type `alsamixer` in terminal and then F6. Adjust the mic
+  and speaker volume with the arrow keys. Press escape to exit.
```

Comment:
there's no internal soundcard. the line should be like in https://supercollider.github.io/development/building-beagleboneblack